blocks/newslist/newslist.js

```diff
@@ -37,12 +37,13 @@ function getHumanReadableDate(dateString) {
   const date = new Date(parseInt(dateString, 10));
   // display the date in GMT timezone
   const country = getCountry();
-  return date.toLocaleDateString(getDateLocales(country), {
+  const localedate = date.toLocaleDateString(getDateLocales(country), {
     timeZone: 'GMT',
     year: 'numeric',
     month: 'long',
     day: '2-digit',
   });
+  return localedate.replace(/de/g, '');
```

Comment:
@veejayricana looks like `de` means `of` in pt. Rather than just replacing this for every locale, can you add a condition like 
```
if (country.includes['pt', 'br']) {
   // de means of in pt/br, replacing with empty string
   localedate.replace(/de/g, '');
}
```